mocca-resiliency4j/src/main/java/com/paypal/mocca/client/MoccaResilience4j.java

```diff
@@ -126,5 +129,43 @@ public Builder bulkhead(final Bulkhead bulkhead) {
             this.feignBuilder = feignBuilder.withBulkhead(bulkhead);
             return this;
         }
+
+        /**
+         *
+         * @param fallBack the fallback object to be set to in this builder
+         * @return
+         */
+        public Builder fallback(final Object fallBack) {
```

Comment:
what does fallback mean in this context?  For all scenarios?

Reply:
As this does not involve any filter for a particular exception, the fallback added using this API will be triggered for all exceptions.